codingtovictory/day5/takeHomeChallenge/index.html

```diff
@@ -0,0 +1,13 @@
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+
+    <img src="https://picsum.photos/id/684/600/400" alt="httpsL//picsum.photos">
```

Comment:
Also make sure you now how to utilize the relative file path if you had downloaded an image instead of using the picsum extension

Reply:
Yup and for the alt we want to give a description of what the photo is. I want know what the image is with the description you provided in the `alt` tag @Bs-Sec @OnkurLal